cocos/network/CCDownloader-apple.mm

```diff
@@ -515,7 +515,18 @@ - (void)URLSession:(NSURLSession *)session downloadTask :(NSURLSessionDownloadTa
     NSString *destPath = [NSString stringWithUTF8String:storagePath];
     NSFileManager *fileManager = [NSFileManager defaultManager];
     NSURL *destURL = nil;
-    
+
+    // On iOS 9 a response with status code 4xx(Client Error) or 5xx(Server Error)
+    // might end up calling this delegate method, saving the error message to the storage path
```

Comment:
I used `might` because I have checked whether all the 4xx & 5xx HTTP status code will end up being success, so just in case any other 4xx or 5xx response calls `URLSession:downloadTask:didFinishDownloadingToURL`, they should check the HTTP status just in case, since 4xx & 5xx shall never result in a successful callback